features/security/pom.xml

```diff
@@ -421,6 +421,27 @@
             <artifactId>secure-boot</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.web</groupId>
+            <artifactId>pax-web-api</artifactId>
+            <version>7.2.4.CODICE</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.web</groupId>
+            <artifactId>pax-web-jetty</artifactId>
+            <version>7.2.4.CODICE</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.web</groupId>
+            <artifactId>pax-web-runtime</artifactId>
+            <version>7.2.4.CODICE</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.web</groupId>
+            <artifactId>pax-web-spi</artifactId>
+            <version>7.2.4.CODICE</version>
```

Comment:
I can do this. I know that versions like this were moved to the ddf-parent project and I didn't want to add there because this should only be temporary until we can upgrade to upcoming release of pax web.